src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -2119,7 +2121,7 @@ private void InsertWorkspace(WorkspaceModel ws, ExtraWorkspaceViewInfo viewInfo
 
             CurrentWorkspace.UpdateWithExtraWorkspaceViewInfo(viewInfo, offsetX, offsetY);
 
-            InsertAnnotations(viewInfo.Annotations, offsetX, offsetY);
+            //InsertAnnotations(viewInfo.Annotations, offsetX, offsetY);
```

Comment:
Just want to confirm this is intended, if yes, let's delete this.

Reply:
Sure thing! I was testing out running the functionality without it, and it seems we don't need this part. Thank you for reminding me to clean up, all done now.